src/components/UserInfo.js

```diff
@@ -13,7 +14,15 @@ export default class UserInfo {
   }
 
   setUserInfo(data) {
-    this._title.textContent = data.title;
-    this._description.textContent = data.description;
+    this._title.textContent = data.name;
+    this._description.textContent = data.about;
+  }
+
+  getProfileImage() {
+    return this._profileImage.src
+  }
+  
```

Comment:
You've implemented a separate method for updating the avatar, it's good, because the avatar is controlled by a different form